### PR TITLE
Strip trailing whitespaces

### PR DESCRIPTION
**Pull request by stof**

### CODE REVIEW

**Comment by stof** [comment]
what is your maintenance process ? Are you merging xdebug_2_5 into master to bring 2.5 changes to master as well ? If yes, I will wait until the merge is done to do these changes in master.

Note that I have not stripped trailing whitespaces on test files.

**Comment by derickr** [comment]
Yes, I merge xdebug_2_5 into master after every "bug fix".

File: xdebug.ini [hunk 1]
```diff
@@ -242,7 +242,7 @@
 ;
 ; Only in Xdebug versions > 2.1
 ;
-; Type: string, Default value:
```
Comment (derickr) [inline xdebug.ini]: This is an auto generated file (by https://github.com/xdebug/xdebug.org/blob/master/html/docs/convert.php), so probably better not to touch it directly unless the conversion script is also fixed.

**Comment by stof** [comment]
@derickr can you merge this one in xdebug_2_5 ? Then I could rebase my branch for args_info so that I don't have to fight my IDE to avoid stripping whitespaces again when doing changes in #303

File: xdebug.ini (hunk 1 above)
Comment (stof) [inline xdebug.ini]: reverted

**Comment by stof** [comment]
Indeed, master does not include any additional trailing whitespaces. So no need for a second PR.

**Comment by derickr** [comment]
LGTM!

I'll merge it after the ```ZEND_ARG_INFO``` changes to make sure we don't get conflicts.

**Comment by stof** [comment]
@derickr as I need to work again on the other branch, I would prefer merging this one first, so that I don't have to skip committing the removal of whitespaces done by my IDE.

Btw, the diff won't conflict, as it touches different lines

**Pull request merged**